server/fishtest/static/js/application.js

```diff
@@ -77,9 +77,7 @@ function formatBytes(bytes) {
 }
 
 function handleApplicationThemes() {
-  if (!getCookie("theme")) {
-    setTheme(mediaTheme());
-  }
+  setTheme(getCookie("theme") ?? mediaTheme());
```

Comment:
This passes `null` in case the left side is false, which the function doesn't handle obviously because it only checks for dark and consider anything else as light, you need to make a separate refresh function to call in the else part of the original code if statement, looks so wrong and bugged.